e3sm_to_cmip/__main__.py

```diff
@@ -717,9 +720,40 @@ def _run_info_mode(self):  # noqa: C901
                             break
                     if not has_vars:
                         continue
-                    messages.append(msg)
 
-        if self.output_path is not None:
+                    # We test here against the input "freq", because atmos mon data satisfies BOTH
+                    # CMIP6_day.json AND CMIP6_mon.json, but we only want the latter in the "hand_msg" output.
+                    # The variables-check below is insufficient as vars "hass" and "rlut" have multiple freqs.
+
+                    if self.freq == "mon" and handler['table'] == "CMIP6_day.json":
+                        continue
+                    if self.freq == "day" and handler['table'] == "CMIP6_Amon.json":
+                        continue
+
+                    hand_msg = None
+                    stat_msg = None
+
+                    raw_vars = []
+                    raw_vars.extend(handler["raw_variables"])
+
+                    allpass = True
+                    for raw_var in raw_vars:
+                        if raw_var in ds.data_vars:
+                            continue
+                        allpass = False
+
+                    if allpass:
+                        stat_msg = f"Table={handler['table']}:Variable={handler['name']}:DataSupport=TRUE"
+                        hand_msg = get_handler_info_msg(handler)
+                        messages.append(hand_msg)
+                    else:
+                        stat_msg = f"Table={handler['table']}:Variable={handler['name']}:DataSupport=FALSE"
+                    print_message(stat_msg, status="info")
```

Comment:
```suggestion
                    logger.info(stat_msg)
```
Can we use the `logger` object defined at the top of this module instead of `print_message()`?

Reply:
Also should we still print out `stat_msg` if it is `None`?

Or should it be within the `else: ` statement to avoid printing `None`?